source/manage/command-line-tools.rst

```diff
@@ -111,2701 +103,341 @@ Options
-  -  `mattermost group channel`_ - Manage Mattermost groups linked to a channel
-  -  `mattermost group team`_ - Manage Mattermost groups linked to a team
   -  `mattermost help`_ - Generate full documentation for the CLI
-  -  `mattermost import`_ - Import data
-  -  `mattermost integrity`_ - Check database schema integrity as well as referential integrity of channels, slash commands, webhooks, posts, schemes, sessions, users, and teams
   -  `mattermost jobserver`_ - Start the Mattermost job server
-  -  `mattermost ldap`_ - AD/LDAP related utilities
-  -  `mattermost license`_ - Licensing commands
-  -  `mattermost logs`_ - Display human-readable logs
-  -  `mattermost permissions`_ - Management of the permissions system
-  -  `mattermost plugin`_ - Management of plugins
-  -  `mattermost reset`_ - Reset the database to initial state
-  -  `mattermost roles`_ - Management of user roles
-  -  `mattermost sampledata`_ - Sample data generation
   -  `mattermost server`_ - Run the Mattermost server
-  -  `mattermost team`_ - Management of teams
-  -  `mattermost user`_ - Management of users
   -  `mattermost version`_ - Display version information
-  -  `mattermost webhook`_ - Management of webhooks
 
-mattermost channel
-------------------
+mattermost db
+--------------
 
 Description
-  Commands for channel management.
+  Commands related to the database
 
 Child Commands
-  -  `mattermost channel add`_ - Add users to a channel
-  -  `mattermost channel archive`_ - Archive a channel
-  -  `mattermost channel create`_ - Create a channel
-  -  `mattermost channel delete`_ - Delete a channel
-  -  `mattermost channel list`_ - List all channels on specified teams
-  -  `mattermost channel modify`_ - Modify a channel's public/private type
-  -  `mattermost channel move`_ - Move a channel to another team
-  -  `mattermost channel remove`_ - Remove users from a channel
-  -  `mattermost channel rename`_ - Rename a channel
-  -  `mattermost channel restore`_ - Restore a channel from the archive
-  -  `mattermost channel search`_ -  Search a channel by name
-
-.. _channel-value-note:
-
-.. note::
-  **{channel} value**
-
-  For the *add*, *archive*, *delete*, *remove*, and *restore* commands, you can specify the *{channels}* value by {team}:{channel} using the team and channel URLs, or by using channel IDs. Channel IDs can be obtained via the `API <https://api.mattermost.com/#operation/GetChannelByName>`_ or the `mattermost channel search </manage/mmctl-command-line-tool.html#mmctl-channel-search>`__ command.
-    
-  For example, in the following URL the *{channels}* value is *myteam:mychannel*: ``https://example.com/myteam/channels/mychannel``
-
-  Also, the team and channel names in the URL should be written in lowercase.
-
-mattermost channel add
-~~~~~~~~~~~~~~~~~~~~~~
+  -  `mattermost db downgrade`_ - Roll back database migrations. Requires either an update plan to roll back to, or comma-separated version numbers to be rolled back.
```

Comment:
@sadohert & @streamer45 - It's unclear to me whether Mattermost continues to support the `mattermost db downgrade` CLI command. Please confirm.

Reply:
I'll defer to @isacikgoz  as he'd know best.

Reply:
It's actually freshly added command, so yes we will support it.